eng/pipelines/handlers.yml

```diff
@@ -160,6 +165,9 @@ stages:
                 platform: ${{ BuildPlatform.name }}
             - pwsh: ./build.ps1 --target=dotnet-pack --configuration="Release" --verbosity=diagnostic
               displayName: 'Pack .NET Maui'
+              ${{ if eq(BuildPlatform.name, 'macOS') }}:
+                timeoutInMinutes: 20
+                continueOnError: true
```

Comment:
I think we should not do this here... If it partially goes through, then we might sign broken packages and release with missing ones.

So far, it has not been hanging on this one I don't think.

Reply:
Removed